test/fixtures/config/self_service_user.json

```diff
@@ -1133,7 +1133,7 @@
                 "666"
               ],
               "merchant_details": {
-                "name": "Government Digital Service",
```

Comment:
I think we have existing test with this, so probably it's better do delete the predefined value and then submit in the test